libmscore/scoreElement.cpp

```diff
@@ -195,6 +195,21 @@ int ScoreElement::treeChildIdx(ScoreElement* child) const
     return -1;
 }
 
+//---------------------------------------------------------
+//   scanElements
+//---------------------------------------------------------
+
+void ScoreElement::scanElements(void* data, void (* func)(void*, Element*), bool all)
+{
+    // Recursively apply scanElements to all children
+    // See also Element::scanElements
```

Comment:
This comment applies to the function as a whole (rather than just part of it) so it would be better to put it before the function as a doc-style comment:

```c++
//---------------------------------------------------------
//   scanElements
/// Recursively apply scanElements to all children.
/// See also Element::scanElements.
//---------------------------------------------------------
```

Reply:
yep, done.